README.md

```diff
@@ -17,6 +17,7 @@ To run a local copy of the wiki:
 3. [Fork this repo](https://guides.github.com/activities/forking/), then clone it. **Make sure to do all work on [another branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch).**
 4. Open a command-line window in the directory you just cloned into, then run the command `yarn` to install required packages.
 5. After packages are installed, start the development server with the command `yarn dev`. You can kill the server by closing the terminal or by pressing <kbd>CTRL+C</kbd>
+    1. If you run into an error when trying to use `yarn dev` on Windows, try taking out `yarn.ps1` and `yarnpkg.ps1` in `user/Appdata/Roaming/npm`. 
```

Comment:
Was this the result of searching around on the internet for a solution or from playing around trying to get it working?
If former, could you link the source for confirmation?

Reply:
It was through the internet I found this solution, here is the source:
https://stackoverflow.com/questions/41117421/ps1-cannot-be-loaded-because-running-scripts-is-disabled-on-this-system

It's the second to last comment with "3" up votes and the reason I chose that instead of the other solution is because I'm not really comfortable with powershell to much and I did not want to mess something up security wise. So instead I saw that answer and because all I'm doing is literally cutting those two files out to see if worked, (I was not afraid of of yarn breaking any more because it was already not working) I had nothing to lose.

Reply:
Alright, Thanks! Just wanted a paper trail for stuff like this. 
btw you should go ahead and add yourself to the mapping credits page as mentioned in my earlier comment. Ill merge in afterwards.

Reply:
No problem, 
I updated myself in the credits, it should now be all good to go.

Reply:
Sorry but do I need to make a new pull request?

Reply:
nope since you made and pushed the commit to the same branch it automatically added it to this pr as you can see below